docs/admin/upgrading.rst

```diff
@@ -23,6 +23,25 @@ Advice
 Basic instructions
 ==================
 
+Composer Upgrade
+----------------
+
+If you had your Elgg 2.3 project installed using composer, you can follow this sequence:
+
+.. code-block:: sh
+
+    composer self-update
+    composer global update fxp/composer-asset-plugin
+
+    cd ./path/to/project/root
+    composer require elgg/elgg:~3.0
+    composer update
+    vendor/bin/elgg-cli upgrade async
```

Comment:
Changed all occurences to `vendor/bin/elgg-cli` because that should work for all cases